stdlib/ocaml_operators.mld

```diff
@@ -0,0 +1,100 @@
+Precedence level and associativity of operators
+
+The following table lists the precedence level of all operator classes
+from the highest to the lowest precedence. A few other syntactic constructions
+are also listed as references.
+
+{%latex:
+%
+% Note: the tables below should be kept in sync with the one in
+% manual/manual/refman/expr.etex .
```

Comment:
In the LaTeX output (and maybe in the HTML output?), could there be a link to the manual table (and maybe back)?

Reply:
Do you mean adding a link to the absolute path of the table (https://caml.inria.fr/pub/docs/manual-ocaml/expr.html)? Using a relative path sounds difficult since the stdlib documentation is not always generated along the manual.

Reply:
Indeed, I had forgotten that the two documents are independent. Let's forget about this idea.